From 0a8aea4cac2953deecdcdfadfab82c2e54dc38cb Mon Sep 17 00:00:00 2001
From: Kacper Kasper <kacperkasper@gmail.com>
Date: Wed, 18 Jul 2018 17:07:19 +0200
Subject: Remove wrong check

* On 32-bit Haiku ptrdiff_t is long, not int. This causes
  undefined symbol errors when loading the shared object.

diff --git a/src/RunStyles.cxx b/src/RunStyles.cxx
index 44d6942..06b3852 100644
--- a/src/RunStyles.cxx
+++ b/src/RunStyles.cxx
@@ -309,7 +309,5 @@ void RunStyles<DISTANCE, STYLE>::Check() const {
 
 template class Scintilla::RunStyles<int, int>;
 template class Scintilla::RunStyles<int, char>;
-#if PTRDIFF_MAX != INT_MAX
 template class Scintilla::RunStyles<ptrdiff_t, int>;
 template class Scintilla::RunStyles<ptrdiff_t, char>;
-#endif
-- 
2.16.4

